decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -81,7 +81,7 @@ def extended_navigation_bar(items, max_items: 5)
     end
 
     def organization_colors
-      css = current_organization.colors.each.map { |k, v| "--#{k}: #{v};" }.join
+      css = current_organization.colors.each.map { |k, v| "--#{k}: #{v};--#{k}-rgb: #{v[1..2].hex},#{v[3..4].hex},#{v[5..6].hex};" }.join
```

Comment:
Can you add some docs here explaining why we're generating both variables?

Reply:
@microstudi yes, some docs on the code! The reasoning is 👍

Reply:
@microstudi so then maybe [chroma](https://github.com/jfairbank/chroma) can help here, a Ruby gem for color manipulation? which has a [`.to_rgba` method](https://github.com/jfairbank/chroma#serializing-colors)

```
css = current_organization.colors.each.map { |k, v| "--#{k}: #{v.paint.to_rgba};" }.join
```
:smile: